src/helpViewer/helpProvider.ts

```diff
@@ -47,14 +47,17 @@ export class HelpProvider {
 
         // starts the background help server and waits forever to keep the R process running
         const scriptPath = extensionContext.asAbsolutePath('R/help/helpServer.R');
-        // const cmd = `${this.rPath} --silent --slave --no-save --no-restore -f "${scriptPath}"`;
+        const scriptExpression = (process.platform === 'win32' ? 
+            `"base::source('${scriptPath.replaceAll('\\', '\\\\')}')"` :
```

Comment:
What if we use something like

```typescript
const args = [
            '--silent',
            '--slave',
            '--no-save',
            '--no-restore',
            '-e',
            'source(commandArgs(TRUE))',
            '--args',
            scriptPath
        ];
```

where the argument quotation is handled internally, not sure if it works on Windows?

Reply:
Very elegant and works nicely, thanks!